frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -321,14 +321,16 @@ export default class TableInteractive extends Component<*, Props, State> {
                     className={cx("cellData", { "cursor-pointer": isSortable })}
                     onClick={isSortable && this.setSort.bind(this, column)}
                 >
-                    <Ellipsified> {columnTitle} </Ellipsified>
-                    {isSortable &&
-                        <Icon
-                            className="Icon ml1"
-                            name={sort && sort[0] && sort[0][1] === "ascending" ? "chevronup" : "chevrondown"}
-                            size={8}
-                        />
-                    }
+                    <Ellipsified><span>
```

Comment:
We'll want to keep the nesting consistent so It'd be great to wrap the spans onto their own line and then indent the children accordingly just like normal HTML.